portfolio/src/main/java/com/google/sps/servlets/CSVServlet.java

```diff
@@ -40,36 +40,33 @@
 @WebServlet("/markers")
 public class CSVServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("application/json;");
-
-    ArrayList<CityCoords> coords = new ArrayList<>();
-
-    try{
-        Scanner scanner = new Scanner(getServletContext().getResourceAsStream("/assets/markers.csv"));
+  private static final Logger log = Logger.getLogger(DataServlet.class.getName());
+  private String toSend;
 
+  @Override
+  public void init(){
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream("/WEB-INF/markers.csv"));
```

Comment:
nit: It's a good practice in Java to always call the super method when overriding a method. So in this case, you would add `super.init()`.

Reply:
fixed!